tools/local-storage/__tests__/storage.test.ts

```diff
@@ -0,0 +1,51 @@
+import storage from 'local-storage-fallback'
```

Comment:
If we shim `window.localStorage` we dont need tests

Reply:
See comment ^